tests/api/test_checkout.py

```diff
@@ -434,6 +434,59 @@ def test_checkout_available_shipping_methods(
     assert data["availableShippingMethods"] == [{"name": shipping_method.name}]
 
 
+@pytest.mark.parametrize(
+    "expected_price_type, expected_price, display_gross_prices",
+    (("gross", 13, True), ("net", 10, False)),
+)
+def test_checkout_available_shipping_methods_with_price_displayed(
+    expected_price_type,
+    expected_price,
+    display_gross_prices,
+    monkeypatch,
+    api_client,
+    checkout_with_item,
+    address,
+    shipping_zone,
+    site_settings,
+):
+    shipping_method = shipping_zone.shipping_methods.first()
+    taxed_price = TaxedMoney(net=shipping_method.price, gross=Money(13, "USD"))
+    apply_taxes_to_shipping_mock = mock.Mock()
+    apply_taxes_to_shipping_mock.return_value = taxed_price
+
+    manager_mock = mock.MagicMock(apply_taxes_to_shipping=apply_taxes_to_shipping_mock)
+    monkeypatch.setattr(
```

Comment:
Again. Why not simpler. Example:
`monkeypatch.setattr(ExtensionsManager, "calculate_checkout_subtotal", Mock(return_value=subtotal))`
There is no need to mock function which returns Manager, then mock Manager and then mock `apply_taxes_to_shipping`. You just need to mock one method on ExtensionManager.